packages/opentelemetry-grpc-utils/package.json

```diff
@@ -64,7 +64,7 @@
     "typescript": "4.1.3"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.18.0",
+    "@opentelemetry/api": "0.18.0",
```

Comment:
Hmm, the `^` was there already before the PR to 0.18.0 was merged.
@dyladan  any hints why renovate-bot prefers a fixed dependency here?

Reply:
I think because the API is no longer in the monorepo. Behavior should be restored to the old version by #1981

Reply:
Ah yes, shimmer is also pinned.